### ISSUE TITLE
Triplets

### ISSUE THREAD

**Issue by B08Z** [babybuddy/babybuddy#633]
I'm using baby buddy to monitor the progression of my premature triplets. 

It's been fantastic I can't thank you enough for the effort that must have gone in. 

One idea and it only really affects multiples is that when adding an activity having the option to select all children.

For example when pumping I have to select a single child but the expressed milk will actually be split.

It would be nice to track activities that are not linked to a single child.

Appreciate this is a pretty niche use case though 

Anyway thanks again

**Comment by cdubz** [comment]
Oaf -- that sounds like a lot!

Tracking premature birth date has been discussed in #369. I'm on board with supporting it but the discussion there never quite got finalized.

Re: selecting multiple children for pumping... that would probably be pretty complicated. I wonder if it would make more sense to just not associate pumping with a child? The challenge there is that the app is very much built around activities related to a specific child so either of those changes would likely involve some tricky refactoring.

**Comment by B08Z** [comment]
Yeah....it's a little full on. Found this app and it's been so helpful. 

I had a quick look around at the structure...... clearly didn't spend too long...cos triplets! and agree that selection of all 3 is not great. 

But adding data and not selecting a child also looked complicated as everything is tied to that entity. 


At present we are just using a ghost child for all those things, it's not ideal but works.

Maybe a parent entity could be added that would allow a sub set of activities to be added?

**Comment by billybonks** [comment]
I think it's worthwhile removing the child from the pumping model; at the end of the day, the intake is measured in the feeding model.

@B08Z I hope all has gone well with your triplets
